zipline/src/jvmMain/resources/META-INF/proguard/zipline.pro

```diff
@@ -0,0 +1,36 @@
+### ZIPLINE
+
+-keepclassmembers class app.cash.zipline.QuickJsException { * ; }
+-keepclassmembers interface app.cash.zipline.InterruptHandler { * ; }
+-keepclassmembers interface app.cash.zipline.internal.bridge.CallChannel { * ; }
```

Comment:
This is insufficient as the type is looked up by FQCN: https://github.com/cashapp/zipline/blob/trunk/zipline/native/OutboundCallChannel.cpp#L26
```suggestion
-keep,allowoptimization interface app.cash.zipline.internal.bridge.CallChannel { * ; }
```